src/App.tsx

```diff
@@ -1,9 +1,15 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, getQuery] = useState('');
+
+  const visibleMovies = [...moviesFromServer]
```

Comment:
You don't need array copy here, filter doesn't mutate arr
```suggestion
  const visibleMovies = moviesFromServer
```